tqp/topic_queue_poller.py

```diff
@@ -32,22 +32,24 @@ def _create_queue():
     try:
         return _create_queue()
     except sqs_client.exceptions.QueueNameExists:
-        queue_url = sqs_resource.get_queue_by_name(QueueName=name).url
```

Comment:
Uh, was this just plain wrong?

Reply:
no, i just switched it to use the client instead of the resource... didn't have any particular reason